auth/api/http/domains/requests.go

```diff
@@ -121,32 +127,18 @@ func (req enableDomainReq) validate() error {
 		return apiutil.ErrMissingID
 	}
 
-	return nil
-}
-
-type disableDomainReq struct {
-	token    string
-	domainID string
-}
-
-func (req disableDomainReq) validate() error {
-	if req.token == "" {
-		return apiutil.ErrBearerToken
+	if _, ok := validStatuses[req.Status]; !ok {
+		return svcerr.ErrInvalidStatus
```

Comment:
Return `apiutil.ErrMissingID`
Service errors are only returned on the service layer.